caluma/caluma_form/models.py

```diff
@@ -350,7 +349,7 @@ def last_modified_answer(self):
         """
 
         return (
-            Answer.objects.filter(Q(document=self) | Q(document__family=self))
+            Answer.objects.filter(document__family=self)
```

Comment:
I think this breaks the behavior for table rows - see comment above: 

> For root documents, we want to look through the whole document, **while for row documents only the table-local answer is wanted.**

Reply:
@anehx Potentially, we could solve this by doing `Q(document__id=self) | Q(document__family=self)`, but I'm afraid Django is too smart and will generate the same SQL.

@czosel How often do we need this feature anyway? Exactly in this form? Might be worth the sacrifice, given the performance penalty we pay for that